dev-docs/publisher-api-reference.md

```diff
@@ -1439,8 +1439,11 @@ Additional information of these properties:
 | `endpoint` | Required | URL | Defines the auction endpoint for the Prebid Server cluster |
 | `syncEndpoint` | Required | URL | Defines the cookie_sync endpoint for the Prebid Server cluster |
 
-*Currently supported vendors are: appnexus & rubicon
-*Note - When using defaultVendor option, accountId and bidders properties still need to be defined.
+**Additional Notes on s2sConfig properties**
+
+- Currently supported vendors are: appnexus & rubicon
+- When using `defaultVendor` option, `accountId` and `bidders` properties still need to be defined.
+- If `timeout` is larger than Prebid.js timeout, this timeout will be automatically adjusted to 75% of the Prebid.js's timeout in order to have it fit within the auction process.
```

Comment:
Suggested rewording to clarify:
If the `s2sConfig` timeout is greater than the Prebid.js timeout, the `s2sConfig` timeout will be automatically adjusted to 75% of the Prebid.js timeout in order to fit within the auction process.

Reply:
Thanks for the feedback; made the requested changes.